portfolio/src/main/webapp/script.js

```diff
@@ -27,8 +27,16 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-function getMessage(){
-   fetch('/data').then(response => response.text().then((message) => {
-       document.getElementById('messageContainer').innerText = message;
-   })); 
-}
+function getCommentSection(){
+    fetch('/data').then(response => response.json()).then((comments) => {
+        for(let i=0; i<comments.length; i++){
+            let textAuthor = '<p><b>' + comments[i].author + '</b></p>';
+            let textComment = '<p>' + comments[i].comment + '</p>';
+            let textTimestamp = '<p> Timestamp: ' + comments[i].timestamp + '</p></br><hr>';
+
+            document.getElementById('commentContainer').innerHTML += textAuthor;
```

Comment:
Using innerHTML like this makes it dangerous for someone going to your site, because this could happen:
(1) someone goes to your web site and makes a comment, putting some dangerous HTML
(2) anyone else going to your web site would then see that dangerous HTML

Can you avoid innerHTML and use innerText instead? If you want formatting, you'll need to create the HTML elements manually - see the todo-list example for details.

Reply:
Oh, I did not think about that. I'll look into it and do the necessary changes